src/test/regress/expected/ch_benchmarks_1.out

```diff
@@ -0,0 +1,236 @@
+SET search_path to "ch benchmarks";
```

Comment:
The same command is repeated?

Reply:
it was not intentional, but lets keep it as an interesting test case :)